llms/googleai/googleai_llm.go

```diff
@@ -31,11 +32,18 @@ const (
 	SAFETY    = "safety"
 )
 
-// New creates a new GoogleAI struct.
-func New(ctx context.Context, options ...option.ClientOption) (*GoogleAI, error) {
-	gi := &GoogleAI{}
+// NewGoogleAI creates a new GoogleAI struct.
+func NewGoogleAI(ctx context.Context, opts ...Option) (*GoogleAI, error) {
```

Comment:
Should this just be New to match the other packages?

Reply:
From the discussion on the issue I'm guessing this is making room for the next step, so LGTM. Sorry about the delay here! I got bit by a bad fluish bug (covid test is negative though) and have had to rest a lot the last couple of days. Thanks!

Reply:
The plan is to add something like `NewVertex` that will create the same `*GoogleAI` provider but using Vertex / GCP credentials, so users will select which one they want by invoking the appropriate constructor